CHANGES_UNRELEASED.md

```diff
@@ -22,7 +18,15 @@ Use the template below to make assigning a version number during the release cut
   - Description of the change with a link to the pull request ([#0000](https://github.com/mozilla/application-services/pull/0000))
 
 -->
+## Nimbus
+### What's changed
+  - Fixed a bug where opt-in enrollments in experiments were not preserved when the application is restarted ([#4324](https://github.com/mozilla/application-services/pull/4324))
+  - The nimbus component now specifies the version of the server's api - currently V1. That was done to avoid redirects. ([#4319](https://github.com/mozilla/application-services/pull/4319))
 
-## Tabs
+## Push
+### What's changed
+  - Fixed a bug where we don't delete a client's UAID locally when it's deleted on the server ([#4325](https://github.com/mozilla/application-services/pull/4325))
 
-Tabs has been Uniffi-ed!
+## Tabs
+### ⚠️ Breaking Changes ⚠️
+ - Tabs has been Uniffi-ed! ([#4192](https://github.com/mozilla/application-services/pull/4192))
```

Comment:
I know this isn't your change, but I want to flag it here for maybe fixing up when we cut the release. Changelogs are for consumers, and *especially* in the case of a breaking change we should put our "consumer" had on and think about what the consumer would like to have called out in the changelog. Specifically in this case, I think we should say more words that consumers should expect e.g. some name changes in the API surface because the bindings are now auto-generated. We don't have to list out all the individual changes, but it's useful to equip consumers with some high-level expectations of what they need to watch out for.

Reply:
/cc @skhamis just for visibility if there was anything specific you wanted to add in the changelog entry here.